src/web/app/actions/beambox/font-funcs.ts

```diff
@@ -274,24 +391,6 @@ const setTextPostscriptnameIfNeeded = async (textElement: Element) => {
   }
 };
 
-const getPathAndTransformFromSvg = async (data: any, isFilled: boolean) =>
-  new Promise<{ d: string; transform: string }>((resolve) => {
-    const fileReader = new FileReader();
-    fileReader.onloadend = (e) => {
-      const svgString = e.target.result as string;
-      const dRegex = svgString.match(/ d="([^"]+)"/g);
-      const transRegex = svgString.match(/transform="([^"]+)/);
-      const d = dRegex ? dRegex.map((p) => p.substring(4, p.length - 1))?.join('') : '';
-      const transform = transRegex ? transRegex[1] : '';
-      resolve({ d, transform });
-    };
-    if (isFilled) {
-      fileReader.readAsText(data.colors);
-    } else {
-      fileReader.readAsText(data.strokes);
-    }
-  });
-
 const convertTextToPath = async (
   textElement: Element,
   bbox: { x: number; y: number; width: number; height: number },
```

Comment:
Can `bbox` be removed?
(And `webNeedConnectionWrapper` in `ActionsPanel` for font functions)